### ISSUE TITLE
Add a small badge on elements that contain custom CSS, and allow overviewing all per-element custom CSS in one place.

### ISSUE THREAD

**Issue by asafm7** [elementor/elementor#15484]
The per-element custom CSS is a nice feature, but it is hard to keep track of it and to overview it, and it can lead to confusion.

I suggest two things:
- Add a small badge on elements that contain custom CSS
- Allow overviewing all per-element custom CSS in one place

**Comment by sf-steve** [comment]
Yes, this is an issue, but probably not used by enough people to warrant a change to the main UI.
Perhaps the navigator (already an advanced interface) might be a better place to put the badge/indicator.

**Comment by asafm7** [comment]
Sounds good

**Comment by sanjupanikkar** [comment]
I agree. Just a simple icon indicator in the tab will do the job. If you want to take the idea further, it might be useful if there was some indication displayed near any set of controls that has been modified.

**Comment by asafm7** [comment]
Maybe a badge on the 'Advanced' tab on the widget settings as well?
